### PR TITLE
MF-571 - Add dashboard

### PR DESCRIPTION
**Pull request by darkodraskovic**
Adds a basic dashboard that reads MF version, num of things and num of channels.

### CODE REVIEW

File: ui/src/Thing.elm [hunk 1]
```diff
@@ -1,15 +1,22 @@
 module Thing exposing (Model, Msg(..), Thing, initial, update, view)
 
 import Bootstrap.Button as Button
+import Bootstrap.ButtonGroup as ButtonGroup
+import Bootstrap.CDN exposing (fontAwesome)
```
Comment (blokovi) [inline ui/src/Thing.elm:5]: This Bootstrap.CDN includes old version of fontAwesome (4.7.0), better include latest version (5.7.2) in Main.elm, same way we included external css. Make a fontAwesome function:
```elm
fontAwesome : Html msg
fontAwesome =
    node "link"
        [ rel "stylesheet"
        , href "https://use.fontawesome.com/releases/v5.7.2/css/all.css"
        , attribute "integrity" "sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
        , attribute "crossorigin" "anonymous"
        ]
        []
```
And then just add `fontAwesome` it in the list where is already `mfStylesheet`

File: ui/src/Version.elm [hunk 2]
```diff
@@ -45,17 +42,42 @@ update msg model =
...
 
-view : Model -> Html Msg
-view model =
-    Grid.container []
-        [ Grid.row []
-            [ Grid.col [] [ text model.response ] 
-            ]
-        ]
+
+-- view : Model -> Int -> Int -> Html msg
```
Comment (drasko) [inline ui/src/Version.elm]: Remove dead code.

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by blokovi — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**